bookmarks.html

```diff
@@ -14,99 +14,158 @@
+              expedita ipsum sit suscipit consequatur.
+            </p>
+          </div>
+
+          <ul class="tag-list">
+            <li class="tag-list__item">lorem</li>
+            <li class="tag-list__item">ipsum</li>
+            <li class="tag-list__item">sit</li>
+            <li class="tag-list__item">dolor</li>
+          </ul>
+          <div class="bookmark">
+            <input checked class="hidden" type="checkbox" id="q2b" />
+            <label class="bookmark__btn" for="q2b">
+              <i class="bookmark__btn--empty far fa-bookmark"></i>
+              <i class="bookmark__btn--filled fas fa-bookmark"></i>
+            </label>
+          </div>
+        </li>
+        <li class="card q-card">
+          <div class="q-card__q-container">
+            <h2 class="q-card__title text-primary">Question</h2>
+            <p class="q-card__q-text">
               Lorem ipsum dolor sit amet consectetur adipisicing elit. Veniam,
               excepturi?
             </p>
           </div>
-          <input type="checkbox" id="q2" /><label class="btn-answer" for="q2">
-            <span class="show">show answer</span>
-            <span class="hide">hide answer</span>
+
+          <input class="hidden q-card__btn-ceckbox" type="checkbox" id="q3" />
+          <label class="q-card__btn" for="q3">
+            <span class="q-card__btn-text q-card__btn-text--show">
+              show answer
+            </span>
+            <span class="q-card__btn-text q-card__btn-text--hide">
+              hide answer
+            </span>
           </label>
-          <div class="answer-container">
-            <h2>Answer</h2>
-            <p class="answer">
+          <div class="q-card__answer-container">
+            <h2 class="q-card__title text-secondary">Answer</h2>
+            <p class="q-card__a-text">
               Lorem ipsum dolor sit amet consectetur adipisicing elit. Quae illo
               expedita ipsum sit suscipit consequatur.
             </p>
           </div>
-          <ul class="tag-container">
-            <li>lorem</li>
-            <li>ipsum</li>
-            <li>sit</li>
-            <li>dolor</li>
+
+          <ul class="tag-list">
+            <li class="tag-list__item">lorem</li>
+            <li class="tag-list__item">ipsum</li>
+            <li class="tag-list__item">sit</li>
+            <li class="tag-list__item">dolor</li>
           </ul>
+
           <div class="bookmark">
-            <input type="checkbox" id="q2b" />
-            <i class="page-bookmarks-bm fas fa-bookmark"></i>
+            <input checked class="hidden" type="checkbox" id="q3b" />
+            <label class="bookmark__btn" for="q3b">
+              <i class="bookmark__btn--empty far fa-bookmark"></i>
+              <i class="bookmark__btn--filled fas fa-bookmark"></i>
+            </label>
           </div>
         </li>
       </ul>
     </main>
 
     <nav class="zz-top">
       <dl class="menu">
```

Comment:
I am still not a fan of using `dl` for a `nav` list. Why don't you go with `ul` and `li`?